convert.sh

```diff
@@ -103,9 +103,9 @@ do
 	# set the origin
 	# note : requires to approve your ssh key: https://github.com/settings/ssh
 	git remote add origin git@github.com:${GITHUB_ORG}/${projectName}-wip.git
-	#push it all
-	git push --tags origin master
-
+	#push all branches and tags
+	git push --all origin
+	git push --tags origin
```

Comment:
I have to triple check, but I think that this is yet another good example where Git option names is again really bad. IIRC --all *only push branches*... So I think we'd need both lines to be sure (at worst, they will display "already up-to-date").

The doc isn't indeed crystal clear (http://git-scm.com/docs/git-push).

Anyway, with the current conversion code, I've not even retrieved branches yet. This is mostly because the branches/ directory doesn't follow any standard rule (some directory with the committer name, etc.) and I didn't have the time to analyze them.

BUT yes, if we retrieve branches (and I plan to add a column the csv file for that per project), then it would be better because the current code will only only push master + all the tags.